app/windows/runner_panel.py

```diff
@@ -402,5 +406,9 @@ def finished(self) -> None:
             if "todds" in self.process.program():
                 self.change_progress_bar_color("success")
 
-        # Cleanup process
+        # Cleanup process and show notification
         self.process.terminate()
+        show_information(
+            title="Operation completed",
+            text="Operation completed, you can close this window.",
```

Comment:
```suggestion
            text="Operation completed, you may close this window.",
```

Reply:
i just used what i saw in other apps if u still prefer i will cghange "can" to "may"
![image](https://github.com/user-attachments/assets/52e888e5-7f0c-4356-b9c1-91dbbcb30667)